CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## [2.3.0](https://launchpad.net/mixxx/+milestone/2.3.0)
+## 2.3.1
+* Echo fix: echo added left channel samples to right channel [#4141](https://github.com/mixxxdj/mixxx/pull/4141)
+* Fix bad phase seek when starting from preroll. [lp1930143](https://bugs.launchpad.net/mixxx/+bug/1930143) [#4093](https://github.com/mixxxdj/mixxx/pull/4093)
+* Fix wrong track being recorded in History [lp1933991](https://bugs.launchpad.net/mixxx/+bug/1933991) [#4041](https://github.com/mixxxdj/mixxx/pull/4041)
+* Rekordbox: Handle exception when parsing corrupt .pdb files [lp1933853](https://bugs.launchpad.net/mixxx/+bug/1933853) [#4040](https://github.com/mixxxdj/mixxx/pull/4040)
+
```

Comment:
```suggestion
### Packaging
* Install udev rule file to the correct location automatically in the CMake install step [#4126](https://github.com/mixxxdj/mixxx/pull/4126)
* Various build fixes for FreeBSD [#4122](https://github.com/mixxxdj/mixxx/pull/4122) [#4123](https://github.com/mixxxdj/mixxx/pull/4123) [#4124](https://github.com/mixxxdj/mixxx/pull/4124)
```

Reply:
Do we actually need subsections in small bugfix release? These are only 10 bullet points ;-)

Reply:
I think we should always separate changes affecting packagers into a subsection.